lib/utils/uid.js

```diff
@@ -4,8 +4,7 @@ let counter = 0
  * @link https://github.com/lodash/lodash/blob/4.17.4/lodash.js#L16143
  * @param {string} prefix
  */
-export default function uid(prefix = "") {
+export default function uid(prefix = "bv_") {
     const id = ++counter
```

Comment:
Excellent.

As long as the uids are only used for internal stuff, and not element id's, or other prop defaults, we should be safe on SSR